Cargo.toml

```diff
@@ -16,17 +16,19 @@ features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = ["async_std", "cookie-secure"]
+default = ["fs", "cookie-secure"]
 docs = ["unstable"]
 unstable = []
 hyperium_http = ["http"]
-async_std = [] # "async-std" when it is not default
+async_std = ["fs"]
 cookie-secure = ["cookie/secure"]
+fs = ["async-std"]
```

Comment:
I can remove the "fs" feature and just rely on "async_std", but I generally prefer feature names that communicate the capabilities that are being enabled by that feature. IMO, it makes it easier for the user to decide whether to enable/disable it.

Reply:
I don't feel too strongly either way; happy to have an extra "fs" feature if you think that's a good idea!